wallet/loader.go

```diff
@@ -108,8 +108,37 @@ func (l *Loader) RunAfterLoad(fn func(*Wallet)) {
 func (l *Loader) CreateNewWallet(pubPassphrase, privPassphrase, seed []byte,
 	bday time.Time) (*Wallet, error) {
 
+	defer l.mu.Unlock()
+	l.mu.Lock()
```

Comment:
Hmm usually in practice we _flip_ this order: so we always lock first, as otherwise an early rewind can cause a panic due to unlock of a mutex that isn't locked.

Reply:
Applies to all other instances here as well, likely harmless in practice, but jumped out at me at first glance as the ordering just stuck out a bit.